Disable the wacom plugin since it's too much work to package the dependencies
diff -Nru gnome-settings-daemon-3.3.91.orig/configure.ac gnome-settings-daemon-3.3.91/configure.ac
--- gnome-settings-daemon-3.3.91.orig/configure.ac	2012-03-09 18:14:56.414116186 -0500
+++ gnome-settings-daemon-3.3.91/configure.ac	2012-03-09 18:15:40.994118275 -0500
@@ -245,12 +245,6 @@
 PKG_CHECK_MODULES(COLOR, [colord >= 0.1.9 gnome-desktop-3.0 >= $GNOME_DESKTOP_REQUIRED_VERSION libcanberra-gtk3])
 
 dnl ---------------------------------------------------------------------------
-dnl - wacom
-dnl ---------------------------------------------------------------------------
-
-PKG_CHECK_MODULES(WACOM, [libwacom >= $LIBWACOM_REQUIRED_VERSION x11 xi xtst $GUDEV_PKG gnome-desktop-3.0 >= $GNOME_DESKTOP_REQUIRED_VERSION xorg-wacom])
-
-dnl ---------------------------------------------------------------------------
 dnl - GConf<->GSettings bridge
 dnl ---------------------------------------------------------------------------
 
@@ -568,7 +562,6 @@
 plugins/smartcard/Makefile
 plugins/sound/Makefile
 plugins/updates/Makefile
-plugins/wacom/Makefile
 plugins/xrandr/Makefile
 plugins/xsettings/Makefile
 data/Makefile
@@ -585,7 +578,6 @@
 data/org.gnome.settings-daemon.plugins.orientation.gschema.xml.in
 data/org.gnome.settings-daemon.plugins.updates.gschema.xml.in
 data/org.gnome.settings-daemon.plugins.xrandr.gschema.xml.in
-data/org.gnome.settings-daemon.peripherals.wacom.gschema.xml.in
 data/org.gnome.settings-daemon.plugins.print-notifications.gschema.xml.in
 po/Makefile.in
 ])
diff -Nru gnome-settings-daemon-3.3.91.orig/data/Makefile.am gnome-settings-daemon-3.3.91/data/Makefile.am
--- gnome-settings-daemon-3.3.91.orig/data/Makefile.am	2012-03-09 18:14:56.414116186 -0500
+++ gnome-settings-daemon-3.3.91/data/Makefile.am	2012-03-09 18:16:01.127452552 -0500
@@ -16,7 +16,6 @@
 	org.gnome.settings-daemon.plugins.xsettings.gschema.xml		\
 	org.gnome.settings-daemon.plugins.housekeeping.gschema.xml	\
 	org.gnome.settings-daemon.plugins.print-notifications.gschema.xml	\
-	org.gnome.settings-daemon.peripherals.wacom.gschema.xml		\
 	org.gnome.settings-daemon.plugins.xrandr.gschema.xml
 
 all_schemas = $(gsettings_SCHEMAS)
diff -Nru gnome-settings-daemon-3.3.91.orig/plugins/Makefile.am gnome-settings-daemon-3.3.91/plugins/Makefile.am
--- gnome-settings-daemon-3.3.91.orig/plugins/Makefile.am	2012-03-09 18:14:56.420782852 -0500
+++ gnome-settings-daemon-3.3.91/plugins/Makefile.am	2012-03-09 18:16:30.844120609 -0500
@@ -16,7 +16,6 @@
 	media-keys	\
 	mouse		\
 	sound		\
-	wacom		\
 	xrandr		\
 	xsettings	\
 	$(NULL)
